src/test/java/uk/gov/hmcts/sptribs/common/notification/CancelHearingNotificationTest.java

```diff
@@ -123,12 +137,17 @@ void shouldNotifyRepresentativeWithEmail() {
 
         //Then
         verify(notificationService).sendEmail(any(NotificationRequest.class));
+        verify(notificationHelper).buildEmailNotificationRequest(
+            "testrepr@outlook.com",
```

Comment:
Could you make this consistent to the other tests by changing to data.getCicCase().getRespresentativeEmailAddress()